src/poetry/core/masonry/builders/wheel.py

```diff
@@ -184,7 +184,7 @@ def _build(self, wheel: zipfile.ZipFile) -> None:
 
                     build_dir = self._path / "build"
                     libs: list[Path] = list(build_dir.glob("lib.*"))
-                    if not libs:
+                    if self._editable or not libs:
```

Comment:
nit: pointless to have constructed the build_dir and done the glob if we're just going to duck out anyway

Reply:
Agreed, especially with the explanatory comment below it, it probably makes sense to pull it out into a separate check.

Also...
Looking at it, using the first folder named `lib.*` is actually pretty sketchy, it won't work correctly if there are multiple build targets being created, e.g. if the build is ran for multiple python versions without cleaning the build dir in between - which explains an issue I've ran into. I'll make a separate issue for it (and if I have time later find a fix)